inspire_classifier/serializers.py

```diff
@@ -20,14 +20,16 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-from __future__ import absolute_import, division, print_function
-
 from marshmallow import Schema, fields
 from marshmallow.validate import OneOf
 
 
+class ScoreSchema(Schema):
+    core = fields.Float(attribute='rejected', required=True)
+    non_core = fields.Float(attribute='non_core', required=True)
+    rejected = fields.Float(attribute='core', required=True)
```

Comment:
replying to my earlier comment: no, it won't work. So the class names are not, in fact, configurable, so should not be set in the config.

Reply:
The reason to have configurable class names is that they can be used elsewhere in the application. For example when checking correspondence between the classifier output (which is a np array) and what classes it corresponds to. 
Secondly,  we can use these to make assertions in the tests if we are getting the correct number of classes as output.
Thirdly, this would ease integrating multiple classifiers (for example for arxiv) into the current code. 

The serializers can be hard-coded I think. But I would like to hear what you think about it.